### PR TITLE
Update examples' License

### PR DESCRIPTION
**Pull request by dmadeka**
@SylvainCorlay @ssunkara1 This adds the License for the two csv files in the `data_files` folder

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
@jasongrout

File: LICENSE [hunk 1]
```diff
@@ -252,6 +252,16 @@ OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
 NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
 EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+Examples' Data Files License
+============================
+
+The files 'gdp_per_capita.csv' and 'country_codes.csv' were adapted from
+the World Bank Data Catalog.
+The World Bank: GDP per capita (current US$).
```
Comment (jasongrout) [inline LICENSE]: This isn't descriptive enough for me to find the data set (by typing that name into http://datacatalog.worldbank.org/). The format should be `The World Bank: Dataset name: Data source`, and I think it wouldn't hurt to put the URL for the dataset in parentheses.

**Comment by jasongrout** [comment]
I think the attribution should also be in the example notebooks that use this data.

**Comment by dmadeka** [comment]
I will add the URLs. There is attribution in the notebooks as well, but I will add them for the country codes

File: LICENSE (hunk 1 above)
Comment (dmadeka) [inline LICENSE]: Isn't data source the link itself?

**Comment by dmadeka** [comment]
@jasongrout made the requested changes

File: examples/Marks/Market Map.ipynb [hunk 2]
```diff
@@ -242,6 +242,13 @@
     "market_map.on_hover(hover_handler)\n",
     "display(market_map)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook uses data derived from the World Bank dataset World Bank: Dataset (URL location). See the LICENSE file for more information."
```
Comment (jasongrout) [inline examples/Marks/Market Map.ipynb]: Thanks. Can you fill in the template with the actual data set, etc.?

**Comment by dmadeka** [comment]
@jasongrout Sorry about that

**Comment by dmadeka** [comment]
@jasongrout ping

**Comment by dmadeka** [comment]
As per @jasongrout we're good to merge this and add CC0 licenses for the notebooks

**Pull request merged**